### PR TITLE
 Split OAuth2 into separate sample

### PR DESCRIPTION
**Pull request by fstanis**
Moves changes introduced by #1475 to a separate sample and reverts the comment section one to the original state.

### CODE REVIEW

File: src/30_Advanced/OAuth2_Login.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<!---{
+  "preview": "default",
+  "default": "preview"
+  }--->
+<!--
+## Introduction
+
+This sample showcases how to use [amp-access](https://www.ampproject.org/docs/reference/components/amp-form)
+to allow an OAuth2-based login flow on your AMP pages, such as Google Sign-In.
```
Comment (sebastianbenz) [inline src/30_Advanced/OAuth2_Login.html]: Can you extend the description to list all demonstrated sign flows to make this easier to discover via search engines...

File: src/30_Advanced/OAuth2_Login.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        },
+        "authorizationFallbackResponse": {
+            "error": true,
+            "loggedIn": false
+        }
+    }
+    </script>
+  </head>
+  <body>
+    <!-- In order to view this section, the user needs to be logged in. We use `amp-access` to integrate an OAuth2-based login and to show and hide the login button depending on whether the user is logged in.
```
Comment (sebastianbenz) [inline src/30_Advanced/OAuth2_Login.html]: "logged in"? => `amp-access="NOT loggedIn"`

File: src/30_Advanced/OAuth2_Login.html [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        `on="tap:amp-access.login-google-sign-in"` specifies which action should be taken when clicking on the login button:
+        `login` defines the property inside the `amp-access` json configuration, while `google-sign-in` defines the endpoint. -->
+    <div amp-access="NOT loggedIn" role="button" tabindex="0" amp-access-hide>
+      <h3 class="m1">Please login to view content</h3>
+      <button on="tap:amp-access.login-google-sign-in" class="ampstart-btn m2 caps">Google Login</button>
+      <button on="tap:amp-access.login-facebook-sign-in" class="ampstart-btn m2 caps">Facebook Login</button>
+      <button on="tap:amp-access.login-github-sign-in" class="ampstart-btn m2 caps">GitHub Login</button>
+    </div>
+    <!-- Use amp-form to submit the comment. In this sample, only a single comment can be added as comments are not persisted in the backend.  -->
```
Comment (sebastianbenz) [inline src/30_Advanced/OAuth2_Login.html]: Remove the comments part.

**Comment by fstanis** [comment]
Updated comments.

**Comment by sebastianbenz** [comment]
Thanks!

**Pull request merged**